File: totosuki/ABC/B/290.py
N, K = map(int, input().split())
S = input()
j = 0
for i in range(len(S)):
  if "o" in S[i]:
    K -= 1
    if K == 0:
      print(S[:i+1] + S[i+1:].replace("o", "x"))
      break

File: OCH_Tao/ABC/B/290.js
const [input, S] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
let [N, K] = input.split(" ").map(x => parseInt(x, 10));
let ans = "";
for (let i = 0; i < N; i++) {
  if (S[i] === "o" && K > 0) {
    ans += "o";
    K--;
  } else {
    ans += "x";
  }
}
console.log(ans);